listeners/syncprogress.go

```diff
@@ -0,0 +1,82 @@
+package listeners
+
+import (
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/wallet"
+)
+
+type SyncProgress struct {
+	SyncStatus chan wallet.SyncStatusUpdate
```

Comment:
`SyncStatusChan`. Append `Chan` always (for other listeners in this pkg as well) so when this variable is used in code in other sites, the variable type is quite immediately obvious.